inference-engine/include/vpu/vpu_plugin_config.hpp

```diff
@@ -120,6 +120,21 @@ DECLARE_VPU_MYRIAD_CONFIG_VALUE(PCIE);
 INFERENCE_ENGINE_DEPRECATED("Use InferenceEngine::MYRIAD_USB instead")
 DECLARE_VPU_MYRIAD_CONFIG_VALUE(USB);
 
+/**
+ * @deprecated Use VPU_MYRIAD_CONFIG_KEY(PLATFORM) instead.
+ */
+INFERENCE_ENGINE_DEPRECATED("Use VPU_MYRIAD_CONFIG_KEY(PLATFORM) instead")
+DECLARE_VPU_CONFIG_KEY(PLATFORM);
```

Comment:
Such keys (with this appointment) are already presented in `inference-engine/include/vpu/myriad_plugin_config.hpp`
You return "double-deprecated" option :)

Reply:
Didn't look into this file, thank you )